packages/plugin-nested-docs/README.md

```diff
@@ -135,8 +135,8 @@ You can also extend the built-in `parent` and `breadcrumbs` fields per collectio
 
 ```js
 import { CollectionConfig } from "payload/types";
-import createParentField from "@payloadcms/plugin-nested-docs/fields/parent";
-import createBreadcrumbsField from "@payloadcms/plugin-nested-docs/fields/breadcrumbs";
+import createParentField from "@payloadcms/plugin-nested-docs/src/fields/parent";
```

Comment:
This is not right, we should not be importing from `src`. If this export is no longer available it is likely due to a change in the publish config post-monorepo.